apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java

```diff
@@ -121,6 +129,16 @@ protected void onCreate(Bundle savedInstanceState) {
         /* Start App Center. */
         AppCenter.start(getApplication(), sSharedPreferences.getString(APP_SECRET_KEY, getString(R.string.app_secret)), Analytics.class, Crashes.class, Distribute.class, Push.class);
 
+        /* Attach NDK Crash Handler (if available) after SDK is initialized. */
```

Comment:
Is that going to be the documented steps? If yes we need to make a method for that to simplify.

Reply:
This should only provide them with an example. I don't believe there is going to be any gain from refactoring this into a function when it's really just 1 line of code for most users. This will be explained in documentation so users know, but it's one of the first steps in the breakpad document that most developers will have to read in order to figure out how to incorporate breakpad in their make process if they haven't already for HockeyApp.

Reply:
I still don't understand why customers would access classes not designed for public audience like ErrorLogHelper, we don't want to expose that class.

Every access to ErrorLogHelper must be wrapped into a public static method of Crashes.

Like Crashes.setUpNativeCrashes. Customers must not have to use our internals.

Reply:
I agree that would be much cleaner. Moved this to a Crashes function and updated the MainActivity to reference that instead.